_collections/_news/2024-09-22-css-welcome-week-newsletter-2024.md

```diff
@@ -0,0 +1,95 @@
+
+
+## CS Societies Night — Monday 30th September 7pm-8pm — Lecture Theatre 1, Teaching and Learning Building
+
+Interested in cybersecurity? Game development? VR? If you're interested in other tech-related societies in the uni, come along to CS Societies night to learn about 6 other CS and STEM societies, pizza included!
+
+Societies attending:
+
+- **AFNOM** - The ethical hacking society
+- **GDS** - Game Development Society
+- **oSTEM** - Out in STEM
+- **VGS** - Video Games Society
+- **VR** - Virtual Reality
+- **AlgoSoc** - Algorithmic Programming Society
+- **CSS** - Computer Science Society (that's us!)
+
+You can find their discord servers in the UoB discord hub here: https://discord.gg/9CH8VtfWQr
+
+The rest of the building will be quiet, so feel free to leave at any time to decompress or cool down.
+
+
+## **Mixers**
+
+CSS is running a few mixers where you can get to know other people in the same year as you, whether that be first years, final years, or postgraduates.
+
+Below are the dates, times and locations:
+
+
+- **Final Year Mixer** — Thursday 27th September 4-6pm — CS Atrium, CS Building
+- **Postgraduate Mixer** — Tuesday 1st October 1-3pm — Joe's Bar, UoB Guild
+- **First Year Mixer** — Wednesday 2nd Wednesday 1-2pm — Joe's Bar, UoB Guild
+
+
+For the final year mixer, pizza and soft drinks will be provided, for the postgraduate and first year mixer, Joe's Bar has a variety of food and drink available.
+
+
+## Pub Night — Friday 4th October 7pm-12am — [The Soak, 563-569 Bristol Rd, Selly Oak, Birmingham B29 6AF](https://maps.app.goo.gl/LWfAZ8ZqYb9v11JFA)
+
+Join CSS at The Soak for a night of drinks, socialisation, and fresh mixed grill. We'll be rounding off the end of freshers activities at one of the best pub and grill houses in Selly Oak!
+
+Make sure to bring a form of **Photo ID** (Not student ID!) if you'd like to buy alcoholic drinks, which will be checked at the door.
+
+
+
+As well as the above, AFNOM, oSTEM, and VR will be running their own events as seen on the timetable, join their discord servers from the hub at https://discord.gg/9CH8VtfWQr to learn more!
+
+If you know anyone who'd like to join, send them our way at https://cssbham.com/join so they can sign up!
+
+If you have any questions, or just want to say hi, we'll be at our stand in the CS Atrium on the 24th and the 25th of September, as well as being available on our discord server at https://cssbham.com/discord. Hope to see you around!
+
+
+Rohit J
+Special Events Officer
+CSS Committee
```

Comment:
Final nag - there need to be 2 spaces after each highlighted line here, otherwise markdown renders it in one line rather than multiple. The 2 spaces add breaks to each line!